ravenframework/SupervisedLearning/DynamicModeDecompositionControl.py

```diff
@@ -165,6 +168,14 @@ def _handleInput(self, paramInput):
     ### Extract the Initialization State Variable Names (x). Optional. If not
     ### found, the state is initialized with the initial values in the state field
     self.initStateID = settings.get('initStateVariables')
+    # FIXME 1718
+    try:
+      [el.split("_init")[1] for el in self.initStateID]
+    except IndexError:
+      self.raiseAnError(IndexError, "initStateVariables must be named {stateVariable}_init")
+    varsToCheck = [el.split("_init")[0] for el in self.initStateID]
+    self.initStateID = [self.initStateID[cnt] for cnt, el in enumerate(varsToCheck) if el in self.stateID]
```

Comment:
Should a warning raised if the initState variable not present in self.stateID?

Reply:
there is an error right before this statement. If not present, there will be an error (because the model cannot be evaluated)

Reply:
(added now a more informative one)